src/api/app/models/workflow/step/submit_request.rb

```diff
@@ -42,6 +42,9 @@ def submit_package
     bs_request.save!
 
     Workflows::ScmEventSubscriptionCreator.new(token, workflow_run, scm_webhook, bs_request).call
+    payload = scm_webhook.payload
+    payload[:number] = bs_request.number
+    SCMStatusReporter.new(payload, payload, @token.scm_token, workflow_run, initial_report: true).call
```

Comment:
The inital report already happens in `Token::Workflow`, no need to do this here again. initial report != inital submit request state

Reply:
I see. Thanks for the tip. I dropped the `initial_report: true` parameter.